python/pip_install/extract_wheels/lib/bazel.py

```diff
@@ -39,7 +39,7 @@ def generate_build_file_contents(
 
         filegroup(
             name="{whl_file_label}",
-            srcs=glob(["*.whl"]),
+            srcs=glob(["*.whl"], allow_empty = True),
```

Comment:
Seems okay to me. It could be strange for consumers of the raw wheel files to get less wheels than are in their requirements.txt but I suppose someone who only want to resolve wheels can pass `--binary-only=":all:"